backend-models/service-tests/fileedservice-tests.txt

```diff
@@ -121,14 +121,41 @@ if ($result && $_SESSION['uid'] == 1) {
 Test the user's access to a course (line 39)
 --------------------------------------------------------------------------------------------------------------------------------------------
 
+*********
+* pre-req:
```

Comment:
pre-req is what is needed before the service can execute.

in this case the pre req for "DELFILE" (which is the first service used) is that 
if (checklogin() && $hasAccess) = IS TRUE

what values these need to have is what is part of the pre-req 
![image](https://user-images.githubusercontent.com/102600391/234229681-41265c91-54c3-4f6e-a67a-41f039bb73da.png)
you need to specify what values they must have for the specific service test.
everything that is